src/components/dialogs/contingency-list/explicit-naming/explicit-naming-form.js

```diff
@@ -28,11 +28,15 @@ export const getExplicitNamingSchema = (id) => ({
         })
     ),
 });
-const suppressEnter = (params) => {
+const suppressKeyboardEvent = (params) => {
     const KEY_ENTER = 'Enter';
```

Comment:
You create 5 const but honestly imo it's useless here

```suggestion
    const key = params.event.key
    return key === 'Enter' || key === 'ArrowLeft' || key === 'ArrowRight'
```

Reply:
it's how it is shown in [the documentation](https://www.ag-grid.com/react-data-grid/keyboard-navigation/#suppress-keyboard-events), to use a `const KEY_*`

Reply:
It's because they are reused sometimes. Here I don't see the point. But not a big deal anyway.